src/benchcab/utils/fs.py

```diff
@@ -72,3 +72,19 @@ def mkdir(new_path: Path, **kwargs):
     """
     get_logger().debug(f"Creating {new_path} directory")
     new_path.mkdir(**kwargs)
+
+
+def prepend_path(var: str, path: str, env=os.environ):
+    """Prepend path to environment variable.
+
+    Parameters
+    ----------
+    var : str
+        Name of environment variable.
+    path : str
+        Path to prepend.
+    env : dict
+        Environment dictionary.
+
+    """
+    env[var] = path + (os.pathsep + env[var] if var in env else "")
```

Comment:
Use `env.get(var, "")` here instead of `if...else` ?

Reply:
I've updated the implementation to:
```python
env[var] = os.pathsep.join(filter(None, [path, env.get(var)]))
```
See [dd04dd4](https://github.com/CABLE-LSM/benchcab/pull/278/commits/dd04dd4f467d5cf12eec3345a34aa0c9ce65064b)